xml/System.Diagnostics/DiagnosticSource.xml

```diff
@@ -69,9 +69,9 @@
         <Parameter Name="name" Type="System.String" />
```

Comment:
This needs a type description. Perhaps:
"An abstract class that allows code to be instrumented for production-time logging of rich data payloads for consumption within the process that was instrumented." In the Remarks section, "For more information, see \[DiagnosticSource User's Guide](https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/DiagnosticSourceUsersGuide.md)."

Reply:
@rpetrusha I can make this change, but where should it go? In remarks?